app/views/pages/dashboard.html.erb

```diff
@@ -1,9 +1,14 @@
 <h1 id="dashboard-heading">My Dashboard</h1>
 
 <div class="row">
-  <div class="col-12 col">
+  <% if (@owned_pokemons.count >= 1 && @bookings.count >= 1) %>
+    <%= render 'pages/dashboard_partials/dash-owned-booked', owned_pokemons: @owned_pokemons, owned_pokemons: @owned_pokemons %>
 
-  </div>
+  <% elsif @owned_pokemons.count >= 1 %>
+    <%= render 'pages/dashboard_partials/dash_owned', owned_pokemons: @owned_pokemons %>
 
+  <% else %>
```

Comment:
Add one more condition for possible new user, you have no upcoming bookings and you have no pokemen that you're renting out.